packages/experience-builder-sdk/src/core/sdkFeatures.ts

```diff
@@ -4,4 +4,5 @@ export const sdkFeatures: Record<string, unknown> = {
   hasVersionHistory: true,
   cfVisibility: true,
   patternResolution: true,
+  patternBreakpointDesignValues: true,
```

Comment:
Just by the name, it might be harder to understand for other devs what this is for.
```suggestion
  // [SPA-2602] allow merging the default and overwriting design values for patterns by breakpoint
  patternBreakpointDesignValues: true,
```